phase3/flannel/flannel-cluster-role-binding.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Why not YAML and everything in the same file?

Reply:
Unfortunately, when I added support for weave-net, I came to the same conclusion that kubernetes-anywhere had a strong preference for having everything in json/jsonnet and similarly broke the manifest up into multiple json files. It's not impossible to add a multidoc yaml file for an addon, but it would require some changes to the `phase3/do` script. I wouldn't consider this approach a non-starter (even though I think yaml is easier to read).